README.md

```diff
@@ -1 +1,2 @@
 # hw01
```

Comment:
Hi Sandy! I reviewed your homework and I feel I would have love if you include more about yourself and use other of the tools we saw in classes. When I did my homework, I started with headers and bold as you did. After I tried ordered lists (using *) and links (underlining the link). Next time I am planning to take more chances and include images or graphics. I am sure you will try new things over the time of the course!